joblib/test/test_parallel.py

```diff
@@ -10,6 +10,10 @@
 import sys
 import io
 import os
+import subprocess
+from threading import Timer
+import re
```

Comment:
Well, that's not critical but that's a pep8 : https://www.python.org/dev/peps/pep-0008/#imports